packages/harmony/src/components/button/Button/Button.tsx

```diff
@@ -59,37 +28,228 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
     } = props
     const { _isHovered, _isPressed, isLoading } = baseProps
     const isDisabled = disabled || isLoading
+    const {
+      type,
+      color: themeColors,
+      cornerRadius,
+      shadows,
+      spacing,
+      typography
+    } = useTheme()
+
+    // Size Styles
+    const smallStyles: CSSObject = {
+      gap: spacing.unit1,
```

Comment:
wondering if we should prefer `spacing.xs` for gap, margin, and padding 🤔

Maybe not here if we don't want the buttons not to be subject to change with the spacing presets?

Reply:
yea you're right, all of the paddings and gaps and things should be spacing vars